src/tutorials/pet-shop.md

```diff
@@ -375,6 +375,99 @@ Since arrays can only return a single value given a single key, we create our ow
 
 Note the **memory** attribute on `adopters`. The memory attribute tells Solidity to temporarily store the value in memory, rather than saving it to the contract's storage. Since `adopters` is an array, and we know from the first adoption test that we adopted pet `expectedPetId`, we compare the testing contracts address with location `expectedPetId` in the array.
 
+## Testing the smart contract using Javascript 
+
+Truffle is very flexible when it comes to smart contract testing, in that tests can be written either in JavaScript or Solidity. In this tutorial, we'll be writing our tests in Javascript using chai and mocha library.
+
+1. Create a new file named `TestAdoption.test.js` in the `test/` directory.
```

Comment:
So I think I would rather this test file be named `testAdoption.test.js` as I think it is more common to use Pascal case for files that export classes.

Reply:
I wonder also if we should remove the `.test` portion of the filename. None of our other boxes use this convention for the JS tests. Do you feel strongly either way about it?

Reply:
It's not necessary to put `.test` within the fileName however, when one reads their code, it is easier to differentiate between the test files from the non-test files.

Reply:
Sure, I'm more concerned about the fact that it breaks from convention that we use elsewhere. But if you want to keep it in then that is ok with me!